Gemfile.lock

```diff
@@ -1,119 +1,120 @@
 PATH
   remote: .
   specs:
-    log_book (0.3.1)
+    log_book (0.5.2)
       acts-as-taggable-on
       rails
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (5.0.2)
-      actionpack (= 5.0.2)
-      nio4r (>= 1.2, < 3.0)
+    actioncable (5.1.5)
```

Comment:
I had to update ActiveSupport otherwise the new method saved_changes was not yet included and tests were not passing